### PR TITLE
updated language options

### PR DESCRIPTION
**Pull request by rehan99000**
**Description:**
Update `LANGUAGES` from edx-platform.

### CODE REVIEW

File: ecommerce/settings/base.py [hunk 1]
```diff
@@ -75,11 +75,89 @@
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#use-i18n
 USE_I18N = True
 
-LANGUAGES = (
-    ('en', _('English')),
-    ('es', _('Spanish')),
-    ('es-419', _('Spanish (Latin American)')),
-)
+LANGUAGES = [
```
Comment (asadmanzoor93) [inline ecommerce/settings/base.py:78]: Use the same format as previous in Ecommerce.

File: ecommerce/settings/base.py [hunk 2]
```diff
@@ -75,11 +75,89 @@
...
+    ('en', u'English'),
+    ('rtl', u'Right-to-Left Test Language'),
+    ('eo', u'Dummy Language (Esperanto)'),  # Dummy languaged used for testing
+    ('fake2', u'Fake translations'),        # Another dummy language for testing (not pushed to prod)
+
+    ('am', u'አማርኛ'),  # Amharic
```
Comment (zubair-arbi) [inline ecommerce/settings/base.py:84]: Why we are providing all languages? We are only supporting English, Arabic and Mangolian for now.

Comment (rehan99000) [reply ecommerce/settings/base.py:84]: All these languages are present in edx-platform `common.py` file. It's a one time thing we won't have to update this again and again in future

File: ecommerce/settings/base.py (hunk 1 above)
Comment (rehan99000) [reply ecommerce/settings/base.py]: done

**Comment by zubair-arbi** [comment]
I guess there is no need for adding all languages. We are only supporting English, Arabic and Mongolian and we can explicitly add languages in future as we keep adding more language support.

@ziafazal your thoughts on this.

**Comment by ziafazal** [comment]
@zubair-arbi I think values of this setting should be same for both edx-platform and ecommerce. If we are supporting all these languages in edx-platform then we should support in ecommerce too.

**Pull request merged**